src/components/color-select/color-select.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import styles from './color-select.module.scss';
+
+export interface ColorSelectOption {
+    colorName: string;
+    color: string;
```

Comment:
`{name: string, value: string}`

Reply:
that is not a value. that is css representation of color.

colorName (comes from ProductChoice.description) is used as a value and display value in SDK. color (comes from ProductChoice.value) is used to render that color

Reply:
what does it mean 'css representation of color'
I can see `backgroundColor: o.color` below.. what is `o.color` if not a color value?

Reply:
if property name `colorName` comes from SDK doesn't mean we can't rename it in scope of the `ColorSelect` component. There is no need adding prefix "color" to each property in the `ColorSelectOption`

Reply:
renamed to `name` and `hexValue` as discussed